src/Microsoft.PowerApps.TestEngine.Tests/TestEngineTests.cs

```diff
@@ -308,7 +308,6 @@ private void SetupMocks(string outputDirectory, TestSettings testSettings, TestS
         [Theory]
         [InlineData(null, "Default-EnvironmentId", "a01af035-a529-4aaf-aded-011ad676f976", "apps.powerapps.com")]
         [InlineData("C:\\testPlan.fx.yaml", "", "a01af035-a529-4aaf-aded-011ad676f976", "apps.powerapps.com")]
-        [InlineData("C:\\testPlan.fx.yaml", "Default-EnvironmentId", "00000000-0000-0000-0000-000000000000", "apps.powerapps.com")]
```

Comment:
Why removing this test? Isn't it good to have this test ?

Reply:
Discussed offline and looks like this scenario is covered in another unittest and not needed here anymore.